datadog/resource_datadog_integration_aws_log_collection.go

```diff
@@ -0,0 +1,145 @@
+package datadog
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/zorkian/go-datadog-api"
+)
+
+func resourceDatadogIntegrationAwsLogCollection() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceDatadogIntegrationAwsLogCollectionCreate,
+		Read:   resourceDatadogIntegrationAwsLogCollectionRead,
+		Update: resourceDatadogIntegrationAwsLogCollectionUpdate,
+		Delete: resourceDatadogIntegrationAwsLogCollectionDelete,
+		Exists: resourceDatadogIntegrationAwsLogCollectionExists,
+		Importer: &schema.ResourceImporter{
+			State: resourceDatadogIntegrationAwsLogCollectionImport,
+		},
+
+		Schema: map[string]*schema.Schema{
+			"account_id": {
+				Type:     schema.TypeString,
+				Required: true,
+				ForceNew: true,
+			},
+			"services": {
+				Type:     schema.TypeList,
+				Required: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+		},
+	}
+}
+
+func resourceDatadogIntegrationAwsLogCollectionExists(d *schema.ResourceData, meta interface{}) (b bool, e error) {
+	// Exists - This is called to verify a resource still exists. It is called prior to Read,
+	// and lowers the burden of Read to be able to assume the resource exists.
+	client := meta.(*datadog.Client)
+
+	logCollections, err := client.GetIntegrationAWSLogCollection()
+	if err != nil {
+		return false, err
```

Comment:
Please use the `translateClientError` method to sanitize client error messages. `translateClientError(err, "error getting aws integration log collection")`. See examples in other resources.